orc8r/cloud/go/services/tenants/servicers/protected/servicer_test.go

```diff
@@ -161,6 +169,16 @@ func TestControlProxyTenantsServicer(t *testing.T) {
 	assert.NoError(t, err)
 	test_utils.AssertMessagesEqual(t, controlProxy, &sampleGetControlProxyRes2)
 
+	// get control_proxy from network ID
+	controlProxy, err = srv.GetControlProxyFromNetworkID(context.Background(), &tenant_protos.GetControlProxyFromNetworkIDRequest{NetworkID: "network_1"})
+	assert.NoError(t, err)
+	test_utils.AssertMessagesEqual(t, controlProxy, &sampleGetControlProxyRes2)
+
+	// get control_proxy from network ID, no tenant
+	_, err = srv.GetControlProxyFromNetworkID(context.Background(), &tenant_protos.GetControlProxyFromNetworkIDRequest{NetworkID: "network_3"})
```

Comment:
Then we could use a value like "network_nonexistent" here to make it more explicit.

Reply:
Done.